src/tree/AksClusterTreeItem.ts

```diff
@@ -0,0 +1,41 @@
+import { AzExtParentTreeItem, AzureTreeItem, ISubscriptionContext } from "vscode-azureextensionui";
+import { Resource } from "azure-arm-storage/lib/models";
+import { CloudExplorerV1 } from "vscode-kubernetes-tools-api";
+import { SubscriptionClient } from 'azure-arm-resource';
+import { toSubscription } from "../azure-api-utils";
+
+export default class AkClusterTreeItem extends AzureTreeItem {
```

Comment:
Actually... I would prefer `Node` to `Item`, because `vscode.TreeItem` refers specifically to the visual rendering of a data object in the tree, and if the data object calls itself a `TreeItem` then I find it harder to be clear whether the code is talking about the data object or the rendering.  But the standard library uses the `Item` suffix for the base type, so if that is the standard then I'm happy to go along with it!

Reply:
While the VS Code API does keep a strict separation between the model and the rendered tree item that represents it, the Azure UI extension does not (I assume for the sake of simplicity).  So, I'd tend to its `TreeItem` naming convention.

Good catch on the missing `s` in the class name, though.  (I tend to follow .NET naming guidelines, hence the `AksCluster` rather than `AKSCluster`; let me know if you'd really prefer the latter.)